### PR TITLE
[OnnxModelLoader, Interpreter Backend] Add support for Sign Onnx Operator

### PR DESCRIPTION
**Pull request by vbaddi**
Summary: Adding support for Sign Onnx Operator, Also Adding Interpreter Backend implementation

Test Plan: Ninja Test is run and added OnnxImporterTest case and OperatorTest case

### CODE REVIEW

**Comment by vbaddi** [comment]
@jfix71 could you please review this change?

File: lib/Importer/ONNXModelLoader.cpp [hunk 1]
```diff
@@ -3814,6 +3814,17 @@ Error ONNXModelLoader::loadMFCC(const ONNX_NAMESPACE::NodeProto &op,
   return Error::success();
 }
 
+Error ONNXModelLoader::loadSign(const ONNX_NAMESPACE::NodeProto &op,
+                                const ArgumentDictionaryTy &dict) {
+  const std::string &opName = loadOperatorName(op);
+  NodeValue in;
+  ASSIGN_VALUE_OR_RETURN_ERR(in, getNodeValueByName(op.input(0)));
+  auto outTy = mod_.uniqueType(*(in.getType()));
+  Node *node = G_->createSign(opName, outTy, in);
```
Comment (jfix71) [inline lib/Importer/ONNXModelLoader.cpp:3823]: We're always using the same input and output type, so seems reasonable to just use the API that does that:
```suggestion
  Node *node = G_->createSign(opName, in);
```

File: tests/unittests/OperatorTest.cpp [hunk 2]
```diff
@@ -6893,6 +6893,44 @@ TEST_P(OperatorTest, Floor_Int8QTy) {
   EXPECT_EQ(outH.raw(4), 2);
 }
 
+TEST_P(OperatorTest, Sign_FloatTy) {
+  CHECK_IF_ENABLED();
+  auto *inp = mod_.createPlaceholder(ElemKind::FloatTy, {3}, "inp", false);
+  bindings_.allocate(inp)->getHandle<float>() = {-1.0, 0.0, 1.0};
+  auto OT = mod_.uniqueType(*(inp->getType()));
+  auto *node = F_->createSign("Sign", OT, inp);
```
Comment (jfix71) [inline tests/unittests/OperatorTest.cpp:6901]: Same as other:
```suggestion
  auto *node = F_->createSign("Sign", inp);
```

File: tests/unittests/OperatorTest.cpp [hunk 3]
```diff
@@ -6893,6 +6893,44 @@ TEST_P(OperatorTest, Floor_Int8QTy) {
...
+TEST_P(OperatorTest, Sign_Int8QTy) {
+  CHECK_IF_ENABLED();
+
+  auto qParams = glow::quantization::chooseQuantizationParams({-100, 100});
+  auto *inp = mod_.createPlaceholder(ElemKind::Int8QTy, {3}, qParams.scale,
+                                     qParams.offset, "input", false);
+  bindings_.allocate(inp)->getHandle<int8_t>() = {-100, 0, 100};
+
+  auto OT = mod_.uniqueType(*(inp->getType()));
+  auto *node = F_->createSign("Sign", OT, inp);
```
Comment (jfix71) [inline tests/unittests/OperatorTest.cpp:6922]: Same as other:
```suggestion
  auto *node = F_->createSign("Sign", inp);
```

File: lib/Importer/ONNXModelLoader.cpp (hunk 1 above)
Comment (vbaddi) [reply lib/Importer/ONNXModelLoader.cpp]: Done

File: tests/unittests/OperatorTest.cpp (hunk 2 above)
Comment (vbaddi) [reply tests/unittests/OperatorTest.cpp]: Done

File: tests/unittests/OperatorTest.cpp (hunk 3 above)
Comment (vbaddi) [reply tests/unittests/OperatorTest.cpp]: Done

**Pull request closed without merging**